modules/calendar/modules.php

```diff
@@ -196,9 +231,9 @@ protected function output() {
             $events = $cal_events->in_date_range($bounds[0], $bounds[1]);
             $out = new Hm_Cal_Output($this, $events);
             $out = $out->output($data, $date, $view);
-            return '<div class="calendar"><div class="content_title">'.$this->trans('Calendar').'</div>'.
-                '<div class="list_controls"><a href="?page=calendar&amp;action=add" title="'.$this->trans('Add Event').'" class="refresh_list">'.
```

Comment:
Classes like list_controls are used in JS context. Did you check all the classes you removed that they are only used for CSS styling and not anywhere in the JS code?

Reply:
Thanks @kroky , 
I took care not to delete the classes that were referenced by the js scripts. For this case, there was no code breakage